### ISSUE TITLE
I get some warnings ... when I run this in ubuntu linux 17.10 and gpu enabled

### ISSUE THREAD

**Issue by josheeg** [bgavran/DNC#5]
python3 main.py
Loaded task
Loaded controller
WARNING:tensorflow:From /home/joshua/Downloads/DNC-master/src/controller_implementations/dnc/memory.py:228: calling softmax (from tensorflow.python.ops.nn_ops) with dim is deprecated and will be removed in a future version.
Instructions for updating:
dim is deprecated, use axis instead
WARNING:tensorflow:From /home/joshua/Downloads/DNC-master/src/controller_implementations/dnc/memory.py:215: calling l2_normalize (from tensorflow.python.ops.nn_impl) with dim is deprecated and will be removed in a future version.
Instructions for updating:
dim is deprecated, use axis instead
This model has 159800 parameters!
2018-03-10 09:46:22.976608: I tensorflow/core/platform/cpu_feature_guard.cc:140] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2018-03-10 09:46:23.107730: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:898] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2018-03-10 09:46:23.108172: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1212] Found device 0 with properties: 
name: GeForce 940M major: 5 minor: 0 memoryClockRate(GHz): 1.176
pciBusID: 0000:04:00.0
totalMemory: 1.96GiB freeMemory: 1.67GiB
2018-03-10 09:46:23.108200: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1312] Adding visible gpu devices: 0
2018-03-10 09:46:23.691006: I tensorflow/core/common_runtime/gpu/gpu_device.cc:993] Creating TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 1426 MB memory) -> physical GPU (device: 0, name: GeForce 940M, pci bus id: 0000:04:00.0, compute capability: 5.0)
Starting...
Summary generated. Step 0  Test cost == 0.696296692 Time == 2.56s
Summary generated. Step 100  Test cost == 0.551536858 Time == 6.58s
Summary generated. Step 200  Test cost == 0.290066570 Time == 7.59s
Summary generated. Step 300  Test cost == 0.282800913 Time == 7.60s

**Comment by bgavran** [comment]
That seems about right, considering the version of TF this was made in. Unless the model acts very strangely and you think this is the reason, I wouldn't worry about it
